core/templates/dev/head/pages/exploration_editor/simple_editor_tab/QuestionIdService.js

```diff
@@ -23,18 +23,18 @@ oppia.factory('QuestionIdService', [function() {
     return string.trim().toLowerCase().split(' ').join('');
   };
   return {
-    SIDEBAR_PREFIX: 'sidebaritem-',
+    SIDEBAR_PREFIX: 'sidebaritem:',
     // Returns an Id used to identify a question subfield.
     getSubfieldId: function(questionId, label) {
-      return [strip(questionId), strip(label)].join('-');
+      return [questionId, strip(label)].join(':');
```

Comment:
Make ':' a named constant, e.g. SIDEBAR_ID_DELIMITER. Use it for SIDEBAR_PREFIX etc.